src/extension.ts

```diff
@@ -100,6 +101,15 @@ export function activate(context: vscode.ExtensionContext) {
 		}
 	});
 
+	// start the watch listener for auto-updates 
+	startListeningForServerChanges();
+
+	// Listener to handle auto-refresh of view - kubectl times out, so we simply restart the watch when it does
+	var watchListener = function restartKubectlListenerOnEvent() {
+		startListeningForServerChanges();
+	};
+	eventEmitter.on('restartKubectlWatch', watchListener);
```

Comment:
what about using a constant for 'restartKubectlWatch' so that in case of a refactor we are sure that all places are changed accordingly